specification/src/main/asciidoc/platform/ApplicationProgrammingInterface.adoc

```diff
@@ -1000,91 +1000,48 @@ _https://jakarta.ee/specifications/xml-web-services/_ .
 
 ===== RMI-JRMP
 
-*TODO Note:* What to do with this section? Leave as-is, mark Optional, remove?
-
 JRMP is the Java technology-specific Remote
 Method Invocation (RMI) protocol. The Jakarta EE security restrictions
 typically prevent all application component types except application
 clients from creating and exporting an RMI object, but all Jakarta EE
 application component types can be clients of RMI objects.
 
-===== RMI-IIOP (Proposed Optional)
-
-*TODO Note:* Since distributed interop is being removed from the EJB spec, do we remove
-this section completely?
-
-The requirements in this section only apply
-to Jakarta EE products that include an Enterprise Beans container and support
-interoperability using RMI-IIOP.
-
-RMI-IIOP allows objects defined using RMI
-style interfaces to be accessed using the IIOP protocol. It must be
-possible to make any remote _enterprise bean accessible via_ RMI-IIOP.
-Some Jakarta  EE products will simply make all remote enterprise beans
-always (and only) accessible via RMI-IIOP; other products might control
-this via an administrative or deployment action. These and other
-approaches are allowed, provided that any remote enterprise bean (or by
-extension, all remote enterprise beans) can be made accessible using
-RMI-IIOP.
-
-Components accessing remote _enterprise
-beans_ may need to use the _narrow_ method of the
-_javax.rmi.PortableRemoteObject_ class, under circumstances described in
-the Jakarta Enterprise Beans specification. Because remote enterprise beans may be deployed
-using other RMI protocols, portable applications must not depend on the
-characteristics of RMI-IIOP objects (for example, the use of the _Stub_
-and _Tie_ base classes) beyond what is specified in the Jakarta Enterprise Beans
-specification.
-
-The Jakarta EE security restrictions typically
-prevent all application component types, except application clients,
-from creating and exporting an RMI-IIOP object. All Jakarta EE application
-component types can be clients of RMI-IIOP objects. Jakarta EE applications
-should also use JNDI to lookup non-Enterprise Beans RMI-IIOP objects. The JNDI names
-used for such non-Enterprise Beans RMI-IIOP objects should be configured at
-deployment time using the standard environment entries mechanism (see
-<<a607, JNDI Naming Context>>).
-The application should fetch a name from JNDI using an environment
-entry, and use the name to lookup the RMI-IIOP object. Typically such
-names will be configured to be names in the COSNaming name service.
-
-This specification does not provide a
-portable way for applications to bind objects to names in a name
-service. Some products may support use of JNDI and COSNaming for binding
-objects, but this is not required. Portable Jakarta EE application clients
-can create non-Enterprise Beans RMI-IIOP server objects for use as callback objects,
-or to pass in calls to other RMI-IIOP objects.
-
-Note that while RMI-IIOP doesn’t specify how
```

Comment:
These security aspects of RMI-IIOP are not referenced in the replacement text for this section.  Is this required?  Or, is it sufficiently covered in the EJB spec (or elsewhere) and this was just redundant?